### ISSUE TITLE
Submit to Maven Central Repo

### ISSUE THREAD

**Issue by kennethjor** [daviddietz/ynab-sdk#2]
Any plans to submit this to the central repo?

**Comment by daviddietz** [comment]
Yes. I finally got back to updating this. I will address the other two issues and finish writing test before submitting to central maven repo
